publish/bin/spark-publish.js

```diff
@@ -39,6 +39,7 @@ const stuckTransactionsCanceller = createStuckTransactionsCanceller({ pgClient:
 console.log(
   'Wallet address:',
   signer.address,
+  // @ts-ignore `signer.address` is declared as `string` but it's actually an `ethers.EthAddress`
```

Comment:
what about instead casting `signer.address` into an `EthAddress`?

Reply:
I managed to solve it. I was trying to do this the whole time:

```javascript
newDelegatedEthAddress(/** @type {import('@glif/filecoin-address').EthAddress} */ signer.address, 'f').toString()
```
... when all I had to do is wrap `signer.address` into parentheses, like this:

```javascript
newDelegatedEthAddress(/** @type {import('@glif/filecoin-address').EthAddress} */ (signer.address), 'f').toString()
```

I am not yet all that familiar with all tricks of JSDoc 😅

Reply:
Yes! This part is so annoying :D